application/controllers/template_controller.py

```diff
@@ -2,22 +2,21 @@
 from structlog import get_logger
 
 from sqlalchemy.exc import SQLAlchemyError
-from flask import current_app
 
 from application.models.models import CommunicationTemplate
 from application.utils.exceptions import InvalidTemplateException, DatabaseError
 from application.models.schema import template_schema
+from application.utils.database import db
 
 
 logger = get_logger()
 
-UPLOAD_SUCCESSFUL = 'The upload was successful'
 PREEXISTING_TEMPLATE = 'ID already exists'
 
 
-def get_template_by_id(template_id, session):
+def get_template_by_id_from_db(template_id):
```

Comment:
Don't think we need the from_db in the method name

Reply:
I wanted to be explicit compared to the get_comms_template_by_id

Reply:
Fair enough I don't think it's detail that needs to be added. I guess you could say would putting API in the method name of all of the flask routes